### PR TITLE
JBPM-8347 - Add execution error listeners

### PR DESCRIPTION
**Pull request by cristianonicolai**
Assembly:
- https://github.com/kiegroup/droolsjbpm-integration/pull/1799
- https://github.com/kiegroup/jbpm/pull/1473
- https://github.com/kiegroup/kie-soup/pull/84
- https://github.com/kiegroup/droolsjbpm-knowledge/pull/379

### CODE REVIEW

File: kie-internal/src/main/java/org/kie/internal/runtime/error/ExecutionErrorListener.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.kie.internal.runtime.error;
```
Comment (mswiderski) [inline kie-internal/src/main/java/org/kie/internal/runtime/error/ExecutionErrorListener.java]: missing license header

Comment (cristianonicolai) [reply kie-internal/src/main/java/org/kie/internal/runtime/error/ExecutionErrorListener.java]: fixed, thanks

File: kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/archetype-post-generate.groovy [hunk 2]
```diff
@@ -86,6 +86,7 @@ kieserver.dmn.enabled=true
 kieserver.jbpm.enabled=true
 kieserver.jbpmui.enabled=true
 kieserver.casemgmt.enabled=true
+#kieserver.prometheus.enabled=true
```
Comment (jstastny-cz) [inline kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/archetype-post-generate.groovy]: Here we're adding the commented line as a convenience to the user, right? if they need it, they just remove the hash sign ...

Comment (cristianonicolai) [reply kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/archetype-post-generate.groovy]: correct

**Pull request merged**